src/style.css

```
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+/*♡*/
+.stars {
+  display: flex;
+  width: 96px;
+  list-style: none;
+}
+
+.stars__star {
+  background: url(images/star.svg) no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background: url("images/star-active.svg") no-repeat;
```

```suggestion
  background: url(/images/star-active.svg) no-repeat;
```
Спробуй без лапок. Зараз в тебе не завантажується зображення активної зірки